test/v2/services/DeparturesServiceSpec.scala

```diff
@@ -229,4 +229,33 @@ class DeparturesServiceSpec
     }
 
   }
+
+  "When Updating departure with departureId and messageType" - {
```

Comment:
You can drop the When here, it breaks with the style of the spec

Reply:
done